lib/adapters/http/index.js

```diff
@@ -225,11 +225,8 @@ function HttpPouch(opts, callback) {
       method: 'GET',
       url: genUrl(host, '')
     }, function (err, result) {
-      /* istanbul ignore next */
-      if (err) {
-        return callback(err);
-      }
-      var uuid = result.uuid + host.db;
+      var uuid = (result && result.uuid) ?
+        result.uuid + host.db : genDBUrl(host, '');
```

Comment:
^ would prefer to wrap `result.uuid + host.db` in parens so it's clear what's going on